docs/apache-airflow/administration-and-deployment/logging-monitoring/metrics.rst

```diff
@@ -227,7 +227,9 @@ Timers
 Name                                                             Description
 ================================================================ ========================================================================
 ``dagrun.dependency-check.<dag_id>``                             Milliseconds taken to check DAG dependencies
+``dagrun.dependency-check``                                      Milliseconds taken to check DAG dependencies. Metric with dag_id tagging.
```

Comment:
I like that you added the tags to this table.   Maybe we should consider revising this list to be more tag-friendly ((not here in this PR, just a thought))

Reply:
Woudl be nice follow-up :)